lib/tasks/bus_stop_field.rb

```diff
@@ -0,0 +1,68 @@
+        { name: 'Schedule holder', column: :schedule_holder, field_type: :choice, choices: ['On pole', 'In shelter', 'None'], description: 'Determine if there is a bus schedule or not.' },
+        { name: 'System map exists', column: :system_map_exists, field_type: :choice, choices: ['New map', 'Old map', 'None'], description: 'Determine if there is a system map at this stop.' },
+        { name: 'Trash', column: :trash, field_type: :boolean, description: "Determine if there is a trash receptacle at the stop or within 10' of the stop." }
+      ],
+      'Accessible' => [
+        { name: 'Accessible', column: :accessible, field_type: :boolean, description: 'Can this stop be safely accessed?' },
+        { name: 'Curb cut', column: :curb_cut, field_type: :choice, choices: ["Within 20'", 'No curb cut', 'No curb'], description: 'Determine the border between the road and the bus stop.' },
+        { name: 'Sidewalk width', column: :sidewalk_width, field_type: :choice, choices: ['More than 36"', 'Less than 36"', 'None'], description: 'Determine if there is a sidewalk at the bus stop location or not. Note the size of the side walk as well.' },
+        { name: 'Bus pull out exists', column: :bus_pull_out_exists, field_type: :boolean, description: 'Determine if there is a formal/constructed pull out in which a bus can pull in and out of.' },
+        { name: 'ADA landing pad', column: :ada_landing_pad, field_type: :boolean, description: "Is there a concrete/brick/asphalt area that is unobstructed and a minimum of 5'x8'?" },
+        { name: 'Obstructions', column: :obstructions, field_type: :choice, choices: ['Yes- tree/branches', 'Yes- bollards/structure', 'Yes - parking', 'Yes - other', 'No'], description: 'Are there any obstructions which make servicing this stop difficult?' }
+      ],
+      'Safety' => [
+        { name: 'Lighting', column: :lighting, field_type: :choice, choices: ["Within 20'", "20' - 50'", 'None'], description: 'Determine if there is lighting in the bus stop.' }
+      ],
+      'Technology' => [
+        { name: 'Solar lighting', column: :solar_lighting, field_type: :boolean, description: 'Determine if this stop has solar lighting.' },
+        { name: 'Has power', column: :has_power, field_type: :choice, choices: ['Yes stub up', 'Yes outlet', 'No'], description: 'Determine if there is power or not.' },
+        { name: 'Real time information', column: :real_time_information, field_type: :choice, choices: ['Yes - solar', 'Yes - power', 'No'], description: 'Determine if this location has a real time information sign.' }
+      ]
+    }
+
+    fields.each_pair do |category, category_fields|
+      category_fields.each.with_index do |category_field, rank|
+        field = Field.create!(name: category_field.fetch(:name),
+                              rank: rank,
```

Comment:
Have you run this task? Does it work?

Is rank `0` a problem with your design?

Reply:
I haven't. And oh it certainly is a problem with my design.

Reply:
Sorry, let me clarify the question. Would it cause a problem, given your design?

Reply:
It works, but rank 0 doesn't make too much sense.

Reply:
Keep in mind that array indices start at 0, is why I bring it up.

The value isn't constrained in the model, so arguably I could set negative rank as well.

Reply:
That's true.

Reply:
I was going to get to model constraints later, but just to be sure I should probably get them done now.